static/app/views/automations/components/automationListRow.tsx

```diff
@@ -28,6 +32,7 @@ export function AutomationListRow({
   const organization = useOrganization();
   const actions = useAutomationActions(automation);
   const {id, name, disabled, lastTriggered, detectorIds = []} = automation;
+  const projectIds = useAutomationProjectIds(automation);
```

Comment:
these will probably need to be the projectSlugs from the ProjectsStore

Reply:
Addressed in 22a010d85cf